app/models/project.rb

```diff
@@ -14,7 +14,7 @@ class Project < ApplicationRecord
   has_many :user_actions
   has_many :data_requests, as: :exportable
 
-  enum rules_applied: [:all_matching_rules, :first_matching_rule]
+  enum status: { paused: 0, active: 1 }
```

Comment:
what happened with the rules_applied enum? Does it need to be removed in the schema?

Reply:
rules can't be defined on a per-project basis, only a per-workflow basis, so this should never have been there. i can add a migration to drop the column if you'd like.

Reply:
migration added